### PR TITLE
docs: add simple deployment guide // add test overview testing

### PR DESCRIPTION
**Pull request by osigge**

### CODE REVIEW

**Comment by marianfoo** [comment]
I have corrected the basic structure.
Unfortunately I cannot give any input on the content
@vobu can you please give feedback here

**Comment by vobu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by marianfoo** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .markdownlint.jsonc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "MD013": false,
+    "MD033": false,
+    // because we're not displaying the front matter "title"
+    "MD025": false
```
Comment (marianfoo) [inline .markdownlint.jsonc:5]: @vobu very good, i thought about to do the same thing

**Pull request merged**